style.css

```diff
@@ -0,0 +1,87 @@
+/* GENERAL */
+
+button {
+  background-color: transparent;
+  border: none;
+}
+
+/* HEADER */
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#header nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5rem 24px;
+  align-items: center;
+}
+
+.icon-md {
+  width: 20px;
+}
+
+#welcome {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+/* HEADLINE SECTION */
+
+#landing {
```

Comment:
- [ ] I can see that you have used the ID selector at almost all the place, kindly let's prefer using class selectors instead since they give you more customization 👍